avro-converter/src/main/java/io/confluent/connect/avro/AvroData.java

```diff
@@ -1402,14 +1509,14 @@ else if (schema.type() == Schema.Type.INT32)
   private static String unionMemberFieldName(org.apache.avro.Schema schema) {
     if (schema.getType() == org.apache.avro.Schema.Type.RECORD ||
         schema.getType() == org.apache.avro.Schema.Type.ENUM) {
-      return splitName(schema.getName())[1];
+      return schema.getFullName();
```

Comment:
So this really is needed, for the case where the name in two nested types is the same, but package is different without this they get treated as the same, when they're not.

How can we add a config flag? Can you help advise or point at an example where done previously?